examples/UberSDK/UberSDKTests/Mocks/UberAuthMocks.swift

```diff
@@ -0,0 +1,63 @@
+///
+/// @Generated by Mockolo
+///
+
+
+
+import AuthenticationServices
+import Foundation
+import UIKit
+import UberCore
+@testable import UberAuth
+
+
+public class AuthProvidingMock: AuthProviding {
+    public init() { }
+
+
```

Comment:
NIT: extra line

Reply:
This is generated code from Mockolo so I'd rather not edit manually.